tests/acceptance/cli/test_mixed_configurations.py

```diff
@@ -63,163 +58,6 @@ def test_destroy_with_no_configurations(click_runner, custom_filepath):
     assert not custom_filepath.exists()
 
 
-def test_coexisting_configurations(click_runner,
-                                   custom_filepath,
-                                   testerchain,
-                                   agency_local_registry,
-                                   mocker):
-    #
-    # Setup
-    #
-
-    if custom_filepath.exists():
-        shutil.rmtree(str(custom_filepath), ignore_errors=True)
-    assert not custom_filepath.exists()
-
-    # Parse node addresses
-    # TODO: Is testerchain & Full contract deployment needed here (causes massive slowdown)?
-    alice, ursula, another_ursula, staking_provider, *all_yall = testerchain.unassigned_accounts
-
-    envvars = {NUCYPHER_ENVVAR_KEYSTORE_PASSWORD: INSECURE_DEVELOPMENT_PASSWORD,
-               NUCYPHER_ENVVAR_ALICE_ETH_PASSWORD: INSECURE_DEVELOPMENT_PASSWORD,
-               NUCYPHER_ENVVAR_BOB_ETH_PASSWORD: INSECURE_DEVELOPMENT_PASSWORD}
-
-    # Future configuration filepaths for assertions...
-    public_keys_dir = custom_filepath / 'keystore' / 'public'
-    known_nodes_dir = custom_filepath / 'known_nodes'
-
-    # ... Ensure they do not exist to begin with.
-
-    # No keys have been generated...
-    assert not public_keys_dir.exists()
-
-    # No known nodes exist...
-    assert not known_nodes_dir.exists()
-
-    # Not the configuration root...
-    assert not custom_filepath.is_dir()
-
-    # ... nothing
-    None
-
-    #
-    # Create
-    #
-
-    # Expected config files
-    alice_file_location = custom_filepath / AliceConfiguration.generate_filename()
-    ursula_file_location = custom_filepath / UrsulaConfiguration.generate_filename()
-
-    # Use a custom local filepath to init a persistent Alice
-    alice_init_args = ('alice', 'init',
-                       '--network', TEMPORARY_DOMAIN,
-                       '--payment-network', TEMPORARY_DOMAIN,
-                       '--eth-provider', TEST_ETH_PROVIDER_URI,
-                       '--pay-with', alice,
-                       '--registry-filepath', str(agency_local_registry.filepath.absolute()),
-                       '--config-root', str(custom_filepath.absolute()))
-
-    result = click_runner.invoke(nucypher_cli, alice_init_args, catch_exceptions=False, env=envvars)
-    assert result.exit_code == 0
-
-    # All configuration files still exist.
-    assert alice_file_location.is_file()
-
-    # Use the same local filepath to init a persistent Ursula
```

Comment:
Should we keep the Ursula aspect of this test, or is similar testing already done separately?

Reply:
Good question -I had a good think about this.  I came to the conclusion that the original intent of this test is to prevent leakage between different types of configurations, and since we no longer expect someone to run (for example) and alice and an ursula on the same host, it's no longer relevant.

Reply:
Indeed, there is good integration and unit test coverage for the config API.  It can't hurt to reintroduce a basic CLI config test for ursula to make sure we're testing that layer.

Reply:
Upon further investigation, there's ample ursula CLI config acceptance testing.